packages/hooks/src/useWindowDimensions/useWindowDimensions.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+
+  return {
+    width,
+    height,
+  };
+}
+
+export function useWindowDimensions() {
```

Comment:
I made this a "real" hook because it's generic enough. it's literally a base hook from React Native so I feel pretty safe in that line of thought

however maybe we aren't quite ready for it to be at this level, since it's literally being used in exactly 1 place right now

I'm open to demoting it down to be a hook within the `Menu/hooks` directory